hooks/node_handlers/base_render_handler.py

```diff
@@ -0,0 +1,372 @@
+import sgtk
+
+import hou
+
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class BaseRenderNodeHandler(HookBaseClass):
+    """
+    Base class for all render output nodes.
+    """
+
+    SGTK_PASS_NAME = "sgtk_pass_name"
+
+    # aovs
+    AOV_COUNT = ""
+    AOV_NAME_TMPL = "{}"
+    AOV_FILE_TMPL = "{}"
+    AOV_USE_FILE_TMPL = "{}"
+
+    # archives
+    ARCHIVE_ENABLED = ""
+    ARCHIVE_OUTPUT = ""
+
+    # templates
+    AOV_WORK_TEMPLATE = "aov_work_template"
+    AOV_PUBLISH_TEMPLATE = "aov_publish_template"
+    ARCHIVE_WORK_TEMPLATE = ""
+    ARCHIVE_PUBLISH_TEMPLATE = ""
+
+    # strings
+    AOV_ERROR = ""
+
+    def _get_template_fields_from(self, node, additional_fields={}):
```

Comment:
Use `None` to avoid issues with Python internally reusing the same dictionary instance `{}`, [See also anti-patterns post here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html) 

```suggestion
    def _get_template_fields_from(self, node, additional_fields=None):
```

Same at line 104